code/game/objects/items/weapons/storage/secure.dm

```diff
@@ -60,7 +60,7 @@
 		return
 
 	// -> storage/attackby() what with handle insertion, etc
-	..()
+	return ..()

```

Comment:
all these storage items' attackby are supposed to return 1 when the item insertion succeeds.